rosdep/python.yaml

```diff
@@ -8580,6 +8580,17 @@ python3-sqlalchemy-utils:
   nixos: [python3Packages.sqlalchemy-utils]
   opensuse: [python3-SQLAlchemy-Utils]
   ubuntu: [python3-sqlalchemy-utils]
+python3-sqlmodel:
+  debian:
+    '*':
+      packages: [python3-sqlmodel]
+    bullseye:
+      pip:
+        packages: [sqlmodel]
+    buster: null
```

Comment:
I think we can just drop `buster` for now, it is only on LTS support in Debian anyway.